plugins/catalog/src/components/CatalogPage/DefaultCatalogPage.tsx

```diff
@@ -103,6 +103,7 @@ export interface DefaultCatalogPageProps {
   emptyContent?: ReactNode;
   ownerPickerMode?: EntityOwnerPickerProps['mode'];
   pagination?: boolean | { limit?: number };
+  filters?: ReactNode;
```

Comment:
Though this works it does mean having to pass in all the filters. If there are suggestions to make this work where you can just add on but also replace the existing filters I'm very open to making those changes

Reply:
Were you looking at making a DefaultFilters to go with this, to support extension of the defaults?

Reply:
Yes, plan to tackle that tomorrow.

Reply:
Brain wasn't working yesterday but I got this done now, hopefully this was what you had in mind?